python_modules/dagster/dagster_tests/cli_tests/test_project_commands.py

```diff
@@ -76,6 +76,29 @@ def test_scaffold_code_location_command_succeeds():
         assert origins[0].loadable_target_origin.module_name == "my_dagster_code.definitions"
 
 
+def test_scaffold_code_location_command_exclude_succeeds():
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        result = runner.invoke(
+            scaffold_code_location_command,
+            # NOTE: wildcard is not working, need to fix
+            [
+                "--name",
+                "my_dagster_code",
+                "--excludes",
+                "setup.cfg",
+                "--excludes",
+                "setup.py",
```

Comment:
In my testing it looks like there is an implicit wildcard before and after the exclude strings.

```suggestion
                "--excludes",
                "setup",
```